src/components/NavigationFolder.vue

```diff
@@ -197,6 +198,15 @@ export default {
 			}
 			return t('mail', 'Loading …')
 		},
+		isUnifiedButOnlyInbox() {
+			if (!this.folder.isUnified) return false
+			return (
+				this.folder.specialUse.includes('inbox') &&
+				this.folder.isUnified &&
+				this.$store.getters.accounts[1] ===
```

Comment:
you could just check if there are more than two accounts :see_no_evil:

Reply:
Oh my, you're right. Obviously you're not getting it if unified inbox is disabled. I was thinking too far. 🙊

Reply:
Could you adjust your patch? :)

Reply:
Yes, sorry for the long wait!

Reply:
I've changed it to check if there's more than two accounts and simplified it a bit by not using `!` and checking `isUnified` double